tests/foreman/cli/test_leapp_client.py

```diff
@@ -24,7 +24,7 @@
 synced_repos = pytest.StashKey[dict]
 
 RHEL7_VER = '7.9'
-RHEL8_VER = '8.9'
```

Comment:
We can skip it for now. We need new leapp version for it.

Reply:
Removed this, you can update it later when you've new leapp version.
Now, as its gone, we don't care about PRT results, we can merge this as is